mrmap/monitoring/tables.py

```diff
@@ -120,13 +123,17 @@ def render_monitoring_run(self, value):
 
     def render_health_state_code(self, value):
         if value == HealthStateEnum.WARNING.value:
-            icon = Tag(tag='i', attrs={"class": [TextColorEnum.WARNING.value, IconEnum.WARNING.value]}, tooltip=_('This is a warning reason.'))
+            icon = Tag(tag='i', attrs={"class": [TextColorEnum.WARNING.value, IconEnum.WARNING.value]},
```

Comment:
we should get rid of django-bootstrap-set. Rendering speed is slow in tables....